releasenotes/README.md

```diff
@@ -31,6 +31,8 @@ These changes are in the Salesforce CLI release candidate. We plan to include th
 
 ------------
 
+* FIX: The `project generate manifest --from-org` command now completes successfuly when run on an org that has over 100K metadata components. As part of this fix we set the default value of the SF_LIST_METADATA_BATCH_SIZE environment variable to `500` to ensure that the command works correctly when an org has thousands of folder metadata components, such as EmailTemplateFolder.  (GitHub issue [#3197](https://github.com/forcedotcom/cli/issues/3197), source-deploy-retrieve PR [#1511](https://github.com/forcedotcom/source-deploy-retrieve/pull/1511))
+
```

Comment:
You might want to mention that it specifically fixes out of memory errors in this scenario.  Also scenarios where the command seems to hang forever.

Reply:
@jshackell-sfdc for reference:
* if the org had +100K components, `sf` would throw an `out of memory` error
* if < 100K components but > 1K folder types, `sf ` would hang (that's what `SF_LIST_METADATA_BATCH_SIZE=500` fixed)